drivers/scheduler/k8s/specs/wordpress-sharedv4-svc/storage.yaml

```diff
@@ -0,0 +1,51 @@
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: portworx-sc-cms-repl3-sharedv4-svc
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  repl: "3"
+  priority_io: "high"
+  sharedv4: "true"
+  sharedv4_svc_type: "ClusterIP"
+  io_profile: "cms"
+allowVolumeExpansion: true
+---
+apiVersion: storage.k8s.io/v1
+kind: StorageClass
+metadata:
+  name: portworx-sc-db-repl3
+provisioner: kubernetes.io/portworx-volume
+parameters:
+  repl: "3"
+  io_profile: "db"
+  priority_io: "high"
+allowVolumeExpansion: true
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: wp-pv-claim-sharedv4
+  labels:
+    app: wordpress
+  annotations:
+    volume.beta.kubernetes.io/storage-class: portworx-sc-cms-repl3-sharedv4-svc
+spec:
+  accessModes:
+    - ReadWriteMany
+  resources:
+    requests:
+      storage: 1Gi
+---
+apiVersion: v1
+kind: PersistentVolumeClaim
+metadata:
+  name: mysql-pvc-1-sharedv4
```

Comment:
SC portworx-sc-db-repl3 is not using sharedv4. So, we should get rid of "sharedv4" in mysql-pvc-1-sharedv4?

Reply:
Torpedo runs with multiple apps and volume types at the same time. I wanted to group all the PVCs used together for an app that tests sharedv4 volume. Let me change this to `mysql-pvc-wordpress` instead.